src/cli/commands/check.js

```diff
@@ -40,7 +40,7 @@ export async function run(
   }
 
   // get patterns that are installed when running `yarn install`
-  const [, rawPatterns] = await install.hydrate();
+  const [, rawPatterns] = await install.hydrate(true);
```

Comment:
It's for this bit:
```js
await this.fetcher.init();
await this.compatibility.init();
```
It seems that if the fetcher is not run beforehand, the compatibility check does not have the os and cpu data for each package to properly do the check.